### ISSUE TITLE
nodejs 8.6.0 points to nodejs.install 8.5.0

### ISSUE THREAD

**Issue by majkinetor** [chocolatey/chocolatey-coreteampackages#853]
I don't know how this happened really. The relevant [AU report](https://gist.github.com/gep13/bd2eaa76f2a9ab739ca0544c502dca6e/a9358a89b872dd47a752dd4b390a5f29edcb944f#nodejs)

```
nodejs - checking updates using au version 2017.9.27.836
URL check
  https://nodejs.org/dist/v8.6.0/node-v8.6.0-x86.msi
  https://nodejs.org/dist/v8.6.0/node-v8.6.0-x64.msi
nuspec version: 8.5.0
remote version: 8.6.0
New version is available
Automatic checksum skipped
Updating files
  $Latest data:
    FileName32      (String)    node-v8.6.0-x86.msi
    FileName64      (String)    node-v8.6.0-x64.msi
    FileType        (String)    msi
    NuspecVersion   (String)    8.5.0
    PackageName     (String)    nodejs
    URL32           (String)    https://nodejs.org/dist/v8.6.0/node-v8.6.0-x86.msi
    URL64           (String)    https://nodejs.org/dist/v8.6.0/node-v8.6.0-x64.msi
    Version         (String)    8.6.0

  nodejs.nuspec
    setting id:  nodejs
updating version: 8.5.0 -> 8.6.0
  nodejs.nuspec
    (\<dependency .+?"nodejs.install" version=)"([^"]+)" = $1"[8.6.0]" 
Setting package description from README.md
Attempting to build package from 'nodejs.nuspec'.
Successfully created package 'C:\projects\chocolatey-coreteampackages\automatic\nodejs\nodejs.8.6.0.nupkg'
Package updated
nodejs 8.6.0 was pushed successfully to https://push.chocolatey.org
```

Commit [shows](https://github.com/chocolatey/chocolatey-coreteampackages/commit/1fd1e88a759f114ef06ad3467df7b997ab745ebb#diff-ef64aea98c1cb561b9c241c4b7b7d0b6) that it really wasn't changed. I tested locally and it behaves as expected...

Perhaps it is relevant that yesterday we went from 0 to 18 AU errors ?!

**Comment by majkinetor** [comment]
The same behavior from git too: https://chocolatey.org/packages/git/2.14.2

**Comment by majkinetor** [comment]
Looks like this is AU bug in muster after automatic readme is added. I set the version to previous one until this is fixed.

**Comment by majkinetor** [comment]
Fixed in https://github.com/majkinetor/au/commit/86d5fb646d3e1d19a9719c46666f532a8f6641d9

**Comment by majkinetor** [comment]
Git fix: 9250c8973c3ad0af1384f1f477de9d88605250c1
